Slider/Assets/Scenes/Village.unity

```diff
@@ -61255,28 +61330,27 @@ PrefabInstance:
       objectReference: {fileID: 0}
     - target: {fileID: 3652435011426292518, guid: eb8cd11432469439eb214403b87b5b92, type: 3}
       propertyPath: conds.Array.data[2].dialogueChain.Array.data[0].dialogue
-      value: Did you hear? The railway in the caves was shut down.
+      value: Did you hear? The railway in the chloro caves was shut down.
```

Comment:
if just saying caves can be lowercase. If using full name should be capitalized properly eg Chloro Caves